### PR TITLE
Update angularJs_Controller_basic

### PR DESCRIPTION
**Pull request by shelbourn**
Adding the most important name to your list. =) BOOM!

### CODE REVIEW

**Comment by shelbourn** [comment]
# HOW DID YOU LEAVE ME OUT?! SO NOT COOL! Hehe

File: angularJs_Controller_basic [hunk 1]
```diff
@@ -25,7 +25,8 @@
         $scope.customers = [
              { name: 'Patrick Moore', city: 'Torrance' },
              { name: 'Matt Brown', city: 'Saint Louis' },
-             { name: 'Mitchelle Magee', city: 'Long Beach' }
```
Comment (shelbourn) [inline angularJs_Controller_basic]: Added a comma to continue the array so that I could add a name to the list.

File: angularJs_Controller_basic [hunk 2]
```diff
@@ -25,7 +25,8 @@
...
+             { name: 'Mitchelle Magee', city: 'Long Beach' },
```
Comment (shelbourn) [inline angularJs_Controller_basic:28]: Actually, here is where I added the comma.

File: angularJs_Controller_basic [hunk 3]
```diff
@@ -25,7 +25,8 @@
...
+             { name: 'Matthew Shelbourn', city: 'Arcadia' }
```
Comment (shelbourn) [inline angularJs_Controller_basic:29]: And here is where I added the most important name.

# =)

**Comment by NicKAnderson038** [comment]
My first big mistake. Adding you! haha

**Pull request merged**